src/StripeTests/Services/Orders/OrderServiceTest.cs

```diff
@@ -38,22 +39,34 @@ public class OrderServiceTest : BaseStripeTest
                 },
             };
 
-            this.updateOptions = new OrderUpdateOptions
+            this.listOptions = new OrderListOptions
```

Comment:
I alphabetized the initialization here to help with autogen and added the new returnOptions